src/lib/components/tablev2/SingleSelectableContent.js

```diff
@@ -117,7 +122,18 @@ export default function SingleSelectableContent({
 
             return (
               <div {...cellProps} className="td">
-                {cell.render('Cell')}
+                {
+                  // if cell.column.cell is defined that means we have a custom cell renderer
+                  // otherwise it should be just a string and we can use the constrainted text component
+                  !cell.column.cell && typeof cell.value === 'string' ? (
+                    <ConstrainedText
```

Comment:
1. IMO, the tooltip is there **only** when we don't have enough space to display all the text in the cell.
   I don't think if it makes sense to have this tooltip in the following case:


2. I think we should have some padding between the ConstrainedText and the next cell.

![image](https://user-images.githubusercontent.com/18453133/146772934-0b4a5a0c-2f78-4e87-9601-117916a52c6c.png)
@Cuervino WDYT?